.config/forest.dic

```diff
@@ -11,3 +11,9 @@ APIs
 CBOR
 codec
 BLAKE2b
+VM
+multisig
+tuple
+mutex
+RPC
+parameterized
```

Comment:
This word should already be in the built-in dictionary. Is it missing for you?

Reply:
It is

Reply:
Hmm, that's a big problem. I'll look into how the dictionaries can be different.

Reply:
Fixed.